readme.md

```diff
@@ -78,7 +100,51 @@ Paradex is a non custodial decentralised trading platform. This means you retain
+
+To call the orderParams endpoint you need to submit the following information
+```
+market - symbol of market
+orderType - 'buy'|'sell'
+price - price in quote currency
+amount - amount of base currency
+expirationDate - expiration date and time of order in ISO 8601 format
+```
+There are some important details regarding the expirationDate. For more info please see the [order expiry](#order-expiry) section below)
+
+Once you have a zrxOrder to sign and a fee quote you can start the order submission process. Firstly you need to sign the zrxOrder. As described above the order process is slightly different from the payload signing. Below you can see an example of how you would sign the zrxOrder and then add the correct fields to the order to allow it to be submitted to the order endpoint. Once you have the full signed order it should go through the same payload signing process required by the other endpoints.
+```
+import BN = require('bn.js');
+import { utils } from '0x.js/lib/src/utils/utils'
+import * as ethUtil from "ethereumjs-util"
+
+let privateKey = '0xabcabcabcabcabcabcabcabcabcabcabcabcabcabcabcabcabcabcabcabcabca'
+// get orderParams from orderParams endpoint in paradex apx
+let orderParams = {.....} // data returned from orderParams endpoint
+
+let hashHex = utils.getOrderHashHex(orderParams.zrxOrder);
+let hashHexBuffer = ethUtil.toBuffer(hashHex);
+let prefix = ethUtil.toBuffer('\u0019Ethereum Signed Message:\n32');
+let hashedHash = ethUtil.sha3(Buffer.concat([prefix, hashHexBuffer]));
+let signature = ethUtil.ecsign(hashedHash, ethUtil.toBuffer(privateKey));
+
+let signedOrder = orderParams.zrxOrder;
+signedOrder['v'] = signature.v;
+signedOrder['r'] = ethUtil.bufferToHex(signature.r);
+signedOrder['s'] = ethUtil.bufferToHex(signature.s);
+signedOrder['feeId'] = orderParams.fee.id;
+```
+
+## Order expiry
+
+All orders must have an valid expiration date. At the time of order placement this has to be between 5 minutes and 2 weeks in the future. The expiration date is the date that the order remains valid to be processed by the 0x contracts on the blockchain. To allow for enough time for the trade to be broadcast to and processed by the ethereum network an order will only remain valid on the Paradex orderbook if it has at least 5 minutes to go till its expiration date. Therefore if you want an order to appear on the orderbook for a minute you would set the expirationDate to be 6 minutes in the future. 
```

Comment:
I'm thinking we may want to document a 10 minute minimum here, since I think we'll be moving to that in production soon.